test/resources/oc.resource

```diff
@@ -100,6 +100,15 @@ Named Deployment Should Be Available
     [Arguments]    ${name}    ${ns}=${NAMESPACE}    ${timeout}=${DEFAULT_WAIT_TIMEOUT}
     Run With Kubeconfig    oc wait -n ${ns} deploy ${name} --for=condition=Available --timeout=${timeout}
 
+Named Daemonset Should Be Available
+    [Documentation]    Wait for a given daemonset's Available condition to be true
+    ...    ${name}    Name of daemonset to wait for
+    ...    ${ns}    Namespace of named daemonset. Defaults to NAMESPACE suite variable
+    ...    ${timeout}    Period of time to wait for daemonset to reach ready status. Default 60s.
+    [Arguments]    ${name}    ${ns}=${NAMESPACE}    ${timeout}=${DEFAULT_WAIT_TIMEOUT}
+    Run With Kubeconfig
+    ...    oc wait -n ${ns} daemonset ${name} --for=jsonpath='{.status.numberAvailable}'=1 --timeout=${timeout}
```

Comment:
Should we look for `.status.numberReady` instead? Functionally its equivalent but `numberReady` is required.

Reply:
It doesn't really matter either way. `numberAvailable` reflects pods that have maintained a "ready" status for `pod.spec.minReadySeconds`, which defaults to zero if unset. It's still a check for ready pods, just with the delay taken into account, should that be set.